charts/argo-cd/README.md

```diff
@@ -82,6 +82,11 @@ Changes in the `CustomResourceDefinition` resources shall be fixed easily by cop
 
 ## Upgrading
 
+### 4.0.0 and above
+
+This helm chart version contains ArgoCD v2.3.0. The Argo CD Notifications and ApplicationSet are part of Argo CD now. You no longer need to install them separately. The Notifications and ApplicationSet components **are bundled into default** Argo CD installation.
```

Comment:
```suggestion
This helm chart version deploys ArgoCD v2.3. The Argo CD Notifications and ApplicationSet are part of ArgoCD now. You no longer need to install them separately. The Notifications and ApplicationSet components **are bundled into default** ArgoCD installation.
```

Reply:
Oh seems that we use both "Argo CD" and "ArgoCD". I copied most of this statement over and it seema that the official naming is "Argo<space>CD".
Maybe we can adapt that in this major change also?

Reply:
Either works, as long we're consistent :)